Build is broken

The build is broken on Travis: https://travis-ci.org/camptocamp/ngeo/builds/65867203.

```
node buildtools/build.js buildtools/ngeo.json dist/ngeo.js

info ol Parsing dependencies

info ol Compiling 391 sources

ERR! compile /home/travis/build/camptocamp/ngeo/src/directives/search.js:71: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile                * @param {jQuery.event} event Event.

ERR! compile                          ^

ERR! compile 

ERR! compile 

ERR! compile /home/travis/build/camptocamp/ngeo/src/directives/search.js:83: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile                * @param {jQuery.event} event Event.

ERR! compile                          ^

ERR! compile 

ERR! compile /home/travis/build/camptocamp/ngeo/src/directives/search.js:95: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile                * @param {jQuery.event} event Event.

ERR! compile                          ^

ERR! compile 

ERR! compile externs/ngeox.js:290: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile  *   cursorchanged: (function(jQuery.event, Object,

ERR! compile                               ^

ERR! compile 

ERR! compile externs/ngeox.js:292: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile  *   selected: (function(jQuery.event, Object,

ERR! compile                          ^

ERR! compile 

ERR! compile externs/ngeox.js:294: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile  *   autocompleted: (function(jQuery.event, Object,

ERR! compile                               ^

ERR! compile 

ERR! compile externs/ngeox.js:314: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile  * @type {function(jQuery.event, Object, TypeaheadDataset)|undefined}

ERR! compile                    ^

ERR! compile 

ERR! compile externs/ngeox.js:320: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile  * @type {function(jQuery.event, Object, TypeaheadDataset)|undefined}

ERR! compile                    ^

ERR! compile 

ERR! compile externs/ngeox.js:326: ERROR - Bad type annotation. Unknown type jQuery.event

ERR! compile  * @type {function(jQuery.event, Object, TypeaheadDataset)|undefined}

ERR! compile                    ^

ERR! compile 

ERR! compile 9 error(s), 0 warning(s)

ERR! compile 

ERR! Process exited with non-zero status, see log for more detail: 9 

make: *** [dist/ngeo.js] Error 1
```

--- a/examples/search.js
+++ b/examples/search.js
@@ -163,7 +163,7 @@ app.SearchController.prototype.createAndInitBloodhound_ =
 
 
 /**
- * @param {jQuery.event} event Event.
+ * @param {jQuery.Event} event Event.
  * @param {Object} suggestion Suggestion.
  * @param {TypeaheadDataset} dataset Dataset.
  * @this {app.SearchController}

--- a/externs/ngeox.js
+++ b/externs/ngeox.js
@@ -287,11 +287,11 @@ ngeox.MeasureEvent.prototype.feature;
  * @typedef {{
  *   opened: (function()|undefined),
  *   closed: (function()|undefined),
- *   cursorchanged: (function(jQuery.event, Object,
+ *   cursorchanged: (function(jQuery.Event, Object,
  *       TypeaheadDataset)|undefined),
- *   selected: (function(jQuery.event, Object,
+ *   selected: (function(jQuery.Event, Object,
  *       TypeaheadDataset)|undefined),
- *   autocompleted: (function(jQuery.event, Object,
+ *   autocompleted: (function(jQuery.Event, Object,
  *       TypeaheadDataset)|undefined)
  * }}
  */
@@ -311,19 +311,19 @@ ngeox.SearchDirectiveListeners.prototype.closed;
 
 
 /**
- * @type {function(jQuery.event, Object, TypeaheadDataset)|undefined}
+ * @type {function(jQuery.Event, Object, TypeaheadDataset)|undefined}
  */
 ngeox.SearchDirectiveListeners.prototype.cursorchanged;
 
 
 /**
- * @type {function(jQuery.event, Object, TypeaheadDataset)|undefined}
+ * @type {function(jQuery.Event, Object, TypeaheadDataset)|undefined}
  */
 ngeox.SearchDirectiveListeners.prototype.selected;
 
 
 /**
- * @type {function(jQuery.event, Object, TypeaheadDataset)|undefined}
+ * @type {function(jQuery.Event, Object, TypeaheadDataset)|undefined}
  */
 ngeox.SearchDirectiveListeners.prototype.autocompleted;
 

--- a/src/directives/search.js
+++ b/src/directives/search.js
@@ -68,7 +68,7 @@ ngeo.searchDirective = function() {
 
           element.on('typeahead:cursorchanged',
               /**
-               * @param {jQuery.event} event Event.
+               * @param {jQuery.Event} event Event.
                * @param {Object} suggestion Suggestion.
                * @param {TypeaheadDataset} dataset Dataset.
                */
@@ -80,7 +80,7 @@ ngeo.searchDirective = function() {
 
           element.on('typeahead:selected',
               /**
-               * @param {jQuery.event} event Event.
+               * @param {jQuery.Event} event Event.
                * @param {Object} suggestion Suggestion.
                * @param {TypeaheadDataset} dataset Dataset.
                */
@@ -92,7 +92,7 @@ ngeo.searchDirective = function() {
 
           element.on('typeahead:autocompleted',
               /**
-               * @param {jQuery.event} event Event.
+               * @param {jQuery.Event} event Event.
                * @param {Object} suggestion Suggestion.
                * @param {TypeaheadDataset} dataset Dataset.
                */

